python/add_public_cloud_reference.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import os
+import logging
+from google.cloud import storage
```

Comment:
Could we use the existing util we have for this?

Reply:
yes duh, sorry i have no idea why i forgot about the gcp utils class!